web/src/features/fbaCalculator/components/HFICell.tsx

```diff
@@ -0,0 +1,67 @@
+import { makeStyles} from '@material-ui/core'
+import React from 'react'
+import FixedDecimalNumberCell from './FixedDecimalNumberCell'
```

Comment:
Please replace with absolute import for consistency: 
```
import FixedDecimalNumberCell from 'features/fbaCalculator/components/FixedDecimalNumberCell'
```

Reply:
Fixed!